tools/apicoverage/testgrid.go

```diff
@@ -31,63 +31,21 @@ const (
 	overallService = "OverallService"
 )
 
```

Comment:
I wonder if moving the remaining two functions to `apicoverage.go` wouldn't be better.